lib/process-services/src/lib/task-list/components/task-form/task-form.component.spec.ts

```diff
@@ -768,4 +769,198 @@ describe('TaskFormComponent', () => {
+            const inputFieldThree = fixture.debugElement.nativeElement.querySelector('#text3');
+
+            expect(activitFormSelector).toBeDefined();
+            expect(inputFieldOne['disabled']).toEqual(true);
+            expect(inputFieldTwo['disabled']).toEqual(true);
+            expect(inputFieldThree['disabled']).toEqual(true);
+        });
+
+        it('Should be able to enable Form fields after claiming a task', async() => {
+            fixture.detectChanges();
+            getBpmLoggedUserSpy.and.returnValue(of(claimedTaskDetailsMock.assignee));
+            getTaskDetailsSpy.and.returnValue(of(claimedTaskDetailsMock));
+            component.taskId = '456';
+            component.ngOnInit();
+            fixture.detectChanges();
+            await fixture.whenStable();
+            fixture.detectChanges();
+
+            const activitFormSelector = element.querySelector('adf-form');
+            const inputFieldOne = fixture.debugElement.nativeElement.querySelector('#text1');
+            const inputFieldTwo = fixture.debugElement.nativeElement.querySelector('#text2');
+            const inputFieldThree = fixture.debugElement.nativeElement.querySelector('#text3');
+
+            expect(activitFormSelector).toBeDefined();
+            expect(inputFieldOne['disabled']).toEqual(false, 'Task Form field is disabled');
+            expect(inputFieldTwo['disabled']).toEqual(false, 'Task Form field is disabled');
+            expect(inputFieldThree['disabled']).toEqual(false, 'Task Form field is disabled');
+        });
+    });
+
+    describe('Task form action buttons', () => {
+
+        it('Task form action status when candidate user is not have a access to the task claimed by another candidate user', async() => {
+            getTaskDetailsSpy.and.returnValue(of(involvedUserTaskForm));
+            component.taskId = 'mock-task-id';
+            fixture.detectChanges();
+            await fixture.whenStable();
+            fixture.detectChanges();
+
+            const saveButton = fixture.debugElement.nativeElement.querySelector('[id="adf-form-save"]');
+            const completeButton = fixture.debugElement.nativeElement.querySelector('#adf-form-complete');
+            const claimButton = fixture.debugElement.query(By.css('[data-automation-id="adf-task-form-claim-button"]'));
+            const releaseButton = fixture.debugElement.query(By.css('[data-automation-id="adf-task-form-unclaim-button"]'));
+
+            expect(saveButton).not.toBeNull();
+            expect(saveButton['disabled']).toBe(false, 'Save button is disabled');
+            expect(completeButton).not.toBeNull();
+            expect(completeButton['disabled']).toBe(true, 'Complete button is disabled');
+            expect(claimButton).toBeNull();
+            expect(releaseButton).toBeNull();
+        });
+
+        it('Task form action button status before claiming a task with form', async() => {
```

Comment:
Should only claim button be enabled before claiming a task with a form

Reply:
Or maybe "should show only the Claim button as enabled ....."